packages/core/src/app/shipping/SingleShippingForm.tsx

```diff
@@ -35,6 +35,7 @@ export interface SingleShippingFormProps {
     deleteConsignments(): Promise<Address | undefined>;
```

Comment:
Do you know what the difference is between SingleShippingForm and MultiShippingForm? And which one LAPG uses?

Reply:
single seems to be the default for embedded merchants - if I'm understanding correctly, multi involves and address selector component. LAPG uses single